packages/styles/src/theme.ts

```diff
@@ -198,6 +198,22 @@ const lightPalette = {
     paper: colors.athensGray,
     paperSecondary: colors.athensGray,
     paperLight: colors.white,
+    paperExtraLight: colors.white,
+    amountField: colors.athensGray,
```

Comment:
The amountField background is used locally inside the AmountField component. I think it doesn't look like a case for extending the palette.